Mulligan.1.html

```diff
@@ -0,0 +1,174 @@
+            which commonly leads to conflict with Raphael.
+            <br>
+            <br>
+            Leonardo was named after the Italian polymath, painter, engineer,
+            inventor, writer, anatomist, and sculptor <a href="https://en.wikipedia.org/wiki/Leonardo_da_Vinci" style="color:dodgerblue">Leonardo da Vinci.</a></span></div>
+
+        
+            <!-- instead of using br tags, try adding padding to the bottom of each section. This is easiest accomplished by wrapping each section in its div tag like htis:
+                <div style="padding-bottom:20px;"> <-- section
+                        <div></div> <-- image
+                        headline
+                        text
+                </div>
+        -->
+            <div id="rcorners3"><img src="donatello.gif" alt="Donatello" class="rounded-corners" style="float:right;width:200px;height:200px;">
+            </div>
+
+            <div style="padding-bottom:100px;"><span class="name2"><b>Donatello</b></span>
+            <hr style="height:0px;border-top: 3px solid plum" />
+        
+            <span class="main">The scientist, inventor, engineer, and technological genius, Donatello
+            wears a purple mask and wields a bo staff. Donatellow is perhaps the least
+            violent turtle, preferring to use his knowledge to solve conflicts, but 
+            never hesitates to defend his brothers. 
+            <br>
+            <br>
+            Donatello is named after the early Renaissance Italian artist and sculptor 
+            from Florence, <a href="https://en.wikipedia.org/wiki/Donatello" style="color:plum">Donatello.</a></span></div>
+
+
+            <div id="rcorners4"><img src="michelangelo.gif" alt="Michelangelo" class="rounded-corners" style="width:200px;height:200px;">
+            </div>
+
+            <div style="padding-bottom:100px;"><span class="name3"><b>Michelangelo</b></span>
+            <hr style="height:0px;border-top: 3px solid orange" />
+            
+            <span class="main">The stereotypical teenager of the team, Michelangelo is a free-spirited,
+            relaxed, and often goofy jokester, and known for his love of pizza. 
+            Michelangelo wears an orange mask and wields a pair of nunchucks. He 
+            provides the comic relief, though he still has an adventurous side. The 
+            least mature of the four Turtles, he shows characteristics of a "surfer" type 
+            and is often depicted with a Southern Californian accent.
+            <br>
+            <br>
+             He is named after the Renaissance painter, sculptor, architect, poet,
+            and engineer, <a href="https://en.wikipedia.org/wiki/Michelangelo" style="color:orange">Michelangelo.</a></span></div>
+
+
+            <div id="rcorners1"><img src="raphael.gif" alt="Raphael" class="rounded-corners" style="width:200px;height:200px;">
+            </div>
+
+            <span class="name4"><b>Raphael</b></span>
```

Comment:
If possible, don't use bold, italics, or underline elements. Always define these in the CSS. if needed, make a special case for the container. This will allow you to change formatting/layout site wide if needed.